packages/db-sqlite/src/types.ts

```diff
@@ -6,13 +6,36 @@ import type { LibSQLDatabase } from 'drizzle-orm/libsql'
 import type {
   AnySQLiteColumn,
   SQLiteInsertOnConflictDoUpdateConfig,
+  SQLiteTable,
   SQLiteTableWithColumns,
   SQLiteTransactionConfig,
 } from 'drizzle-orm/sqlite-core'
 import type { SQLiteRaw } from 'drizzle-orm/sqlite-core/query-builders/raw'
 import type { Payload, PayloadRequest } from 'payload'
 
+type SQLiteSchema = {
+  relations: GenericRelation
+  tables: SQLiteTable
+}
+
+export type SQLiteSchemaHook = (
```

Comment:
Change to an args object. We always regret passing params directly like this when we need to extend the function args. It is better to begin that way.